Libplanet.Tests/Action/WorldTest.cs

```diff
@@ -505,9 +505,7 @@ public virtual void TotalSupplyTracking()
         }
 
         protected static IWorldState GetLatestWorldState(BlockChain blockChain) =>
-            blockChain.Tip.ProtocolVersion < BlockMetadata.SlothProtocolVersion
-                ? blockChain.GetWorldState()
-                : blockChain.GetNextWorldState();
+            blockChain.GetNextWorldState();
```

Comment:
Do result of this method always exists, though on asynchronous condition?
(I know we'll use synchronous method on tests though)

Reply:
Actually, this helper method seems unnecessary.
As for whether it exists, it depends on the surrounding test code.
Since its used in a synchronous manner via `Append()` this should always exist.